web-sdk/README.md

```diff
@@ -50,8 +53,6 @@ fileName:index.html
 </script>
 ```
 
-To integrate **Fidel Web SDK** in your website or mobile app, you need to add the script above in your website or mobile web view. For mobile apps, however, we recommend using the [mobile SDKs](/mobile-sdk).
```

Comment:
Let's keep this, it's very difficult to set up the mobile SDKs in some frameworks.

Reply:
☝️